FirebaseAdmin/FirebaseAdmin/Messaging/AndroidConfig.cs

```diff
@@ -89,26 +56,41 @@ private string PriorityString
                         return null;
                 }
             }
+            set
+            {
+                switch (value)
+                {
+                    case "high":
+                        Priority = Messaging.Priority.High;
+                        return;
+                    case "normal":
+                        Priority = Messaging.Priority.High;
+                        return;
```

Comment:
Seems like there should be some error handling here for other values?

Reply:
+1

Reply:
Done